ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadPartitions.java

```diff
@@ -320,7 +334,7 @@ private void addPartition(boolean hasMorePartitions, AlterTableAddPartitionDesc
   private String getPartitionName(Path partitionMetadataFullPath) {
     //Get partition name by removing the metadata base path.
     //Needed for getting the data path
-    return partitionMetadataFullPath.toString().substring(event.metadataPath().toString().length());
+    return partitionMetadataFullPath.toString().substring(event.metadataPath().toString().length() + 1);
```

Comment:
Can we do a split instead at '/'. This may be error prone

Reply:
Existing method. just changing the index